### PR TITLE
[API-3099] Fix unmarshal error

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pkg/builds/parameters.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func (p *BuildParameters) Rows() [][]string {
 		result = append(result, []string{
 			param.Type(),
 			param.Name,
-			param.Value,
+			param.Value.(string),
```
Comment (hongshibao) [inline pkg/builds/parameters.go]: maybe can use `fmt.Sprintf("%v", param.Value)`  ?

Comment (nguyenqh-visenze) [inline pkg/builds/parameters.go]: this will raise error if `param.Value` is not `string`?

Comment (jianchao2020) [reply pkg/builds/parameters.go:54]: For your information, this is [a way](https://stackoverflow.com/a/27137607) to convert `interface{}` to `string`.

Comment (ghost) [reply pkg/builds/parameters.go]: Thanks. Updated.

Comment (ghost) [reply pkg/builds/parameters.go]: Yes. According to the [doc](https://golang.org/ref/spec#Type_assertions), if the type assertion is false, a run-time panic occurs.

**Pull request merged**